pkg/sql/sqltelemetry/iam.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package sqltelemetry
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/pkg/server/telemetry"
+)
+
+const iamRoles = "iam.roles"
+
+// IAMOption is to be incremented every time a CREATE/ALTER role
+// with an OPTION (ie. NOLOGIN) happens.
+func IAMOption(opName string, option string) telemetry.Counter {
```

Comment:
Would be better to pull the telemetry.inc into the methods here, rather than return a counter -- see sqltelemetry/show.go